scripts/grest-helper-scripts/setup-grest.sh

```diff
@@ -108,6 +108,9 @@
     fi
     get_cron_job_executable "stake-distribution-update"
     install_cron_job "stake-distribution-update" "*/30 * * * *"
+    
+    get_cron_job_executable "pool-history-cache-update"
+    install_cron_job "pool-history-cache-update" "*/10 * * * *"
```

Comment:
Since we will be running the SQL every 10 mins, I would like to have a check in the SQL to make sure the previous run has been completed before starting a new one (epoch transitions is when time stops)

Reply:
makes sense to add checks like this even if the query is fairly quick.